### ISSUE TITLE
Generated "enum body" types have redundant prefixes.

### ISSUE THREAD

**Issue by drlagos** [eqrion/cbindgen#487]
When using the export prefix option, it is added twice to the "enum body" structs:

PrefixEnum_PrefixCase_Body;

The second prefix is not necessary.

**Comment by emilio** [comment]
Seems to be a problem only when generating C, and not C++.

**Comment by drlagos** [comment]
Yes, it only happens with C. Also in C++ the prefixes are rarely useful since namespaces solve the problem much better.
